src/components/views/EditUserPreferencePage.js

```diff
@@ -118,10 +118,10 @@ const EditUserPreferencePage = () =>{
         }
     ];
 
-    const scale = (distance) => {
-        const previousMarkIndex = Math.floor(distance / 9);
+    const scale = (dist) => {
+        const previousMarkIndex = Math.floor(dist / 9);
         const previousMark = distanceMarks[previousMarkIndex];
-        const remainder = distance % 9;
+        const remainder = dist % 9;
```

Comment:
i would also not shorten that. dist could mean a couple of things, e.g. distribution. and just from the context it is not entirely clear that it is a distance

Reply:
ok